api/extensions/redhat/openshift-checker/0.1.0/README.md

```diff
@@ -0,0 +1,122 @@
+# Docker OpenShift Analyzer
```

Comment:
Should be updated ?

Reply:
I suppose we need to create a new release 0.1.1 to update the readme?

Reply:
I think a new version is needed anyway for the license as it's not really clear on the repository as it's mixing ASL 2 and EPL2